core/models.py

```diff
@@ -1315,3 +1333,14 @@ def coordinates_path(url_parts):
     if not os.path.exists(full_path):
         os.makedirs(full_path)
     return os.path.join(full_path, "coordinates.json.gz")
+
+class Copyright(models.Model):
+    uri = models.CharField(max_length=100, primary_key=True)
+    label = models.CharField(max_length=200)
+
+class LccnDateCopyright(models.Model):
+    lccn = models.CharField(max_length=25)
+    start_date = models.IntegerField()
```

Comment:
We should use date fields here unless there's some clear reason to avoid them.  If an invalid string comes in from the TSV, we want to reject it when ingesting the data, not after the rules have been applied to the database.